tours/src/main/kotlin/edu/artic/tours/TourDetailsViewModel.kt

```diff
@@ -136,8 +130,11 @@ class TourDetailsViewModel @Inject constructor(
      * Navigate user to the Map activity in Tour Context.
      */
     fun onClickStartTour() {
-        tourObservable.take(1)
-                .subscribe { tour ->
+        Observables.combineLatest(tourObservable, chosenTranslation.map { it.underlyingLocale() })
+                .take(1)
+                .subscribe { tourWithLocale ->
```

Comment:
You could decompose here with `.subscribeBy { (tour, locale) ->` instead. What do you think?

Reply:
Sure. Looks cleaner. Thanks